examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/run_quant.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+set -x
+
+function main {
+
+  init_params "$@"
+  run_tuning
+
+}
+
+# init params
+function init_params {
+  for var in "$@"
+  do
+    case $var in
+      --topology=*)
+          topology=$(echo $var |cut -f2 -d=)
+      ;;
+      --dataset_location=*)
+          dataset_location=$(echo $var |cut -f2 -d=)
+      ;;
+      --input_model=*)
+          input_model=$(echo $var |cut -f2 -d=)
+      ;;
+       --output_model=*)
+           tuned_checkpoint=$(echo $var |cut -f2 -d=)
+       ;;
+      *)
+          echo "Error: No such parameter: ${var}"
+          exit 1
+      ;;
+    esac
+  done
+
+}
+
+# run_tuning
+function run_tuning {
+    extra_cmd=''
+    batch_size=8
+    approach='static'
+    DATASET_NAME="NeelNanda/pile-10k"
+    tuned_checkpoint="saved_results"
+
+    if [ "${topology}" = "opt_125m_woq_gptq_debug_int4" ]; then
```

Comment:
`opt_125m_woq_gptq_debug_int4` -> `opt_125m_woq_gptq_int4`, cause you don't have a `--gptq_debug` input.